core/benches/banking_stage.rs

```diff
@@ -266,11 +264,11 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
         assert!(r.is_ok(), "sanity parallel execution");
     }
     bank.clear_signatures();
-    let verified: Vec<_> = to_packet_batches(&transactions, PACKETS_PER_BATCH);
+    let verified: Vec<_> = to_arc_packet_batches(&transactions, PACKETS_PER_BATCH);
     let vote_packets = vote_txs.map(|vote_txs| {
-        let mut packet_batches = to_packet_batches(&vote_txs, PACKETS_PER_BATCH);
+        let mut packet_batches = to_arc_packet_batches(&vote_txs, PACKETS_PER_BATCH);
         for batch in packet_batches.iter_mut() {
-            for packet in batch.iter_mut() {
+            for packet in Arc::make_mut(batch).iter_mut() {
```

Comment:
Can we use your `as_mut` helper?

Reply:
The as_mut was to make the shared code generic.

At this point we need the Arc, so not sure there's value using the trait here

Reply:
^wait nvm i misread the file name. this isn't where i thought it was.
Fixing